src/Endpoints/class-analytics-posts-api-proxy.php

```diff
@@ -12,8 +12,11 @@
 
 use stdClass;
 use WP_REST_Request;
+use WP_Error;
 use Parsely\Parsely;
 
+use function Parsely\Utils\get_date_format;
```

Comment:
@mehmoodak What about something like https://github.com/szepeviktor/phpstan-wordpress

(Never used it, just googled. Since you mentioned PHPStan is like TypeScript, it seemed like their might be community typings out there.)

Reply:
@chriszarate we are already using it in our project and it does help a lot while writing code related to WordPress.

But there is a catch here like some WordPress functions does return `mixed` value which PHPStan complains about so for that we do need an additional doc comment to specify the correct type for our usecase.